lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Converters/LMFeedConvertToFeedPost.swift

```diff
@@ -117,6 +122,24 @@ public struct LMFeedConvertToFeedPost {
         }
     }
     
+    public static func convertToWidget(from data: [LMFeedWidgetDataModel]) -> [LMFeedWidgetContentModel]{
+        data.compactMap{ widget in
+            var lmMeta: LMFeedLMMetaContentModel?
+            if let widgetLMMeta = widget.lmMeta{
+                var pollOptions: [LMFeedPollOptionContentModel]?
+                
+                pollOptions = widgetLMMeta.options.map{
+                    option in
+                    return LMFeedPollOptionContentModel(id: option.id, text: option.text, isSelected: option.isSelected, percentage: option.percentage, uuid: option.uuid, voteCount: option.voteCount)
+                }
```

Comment:
shift params to new line

Reply:
AttachmentMeta was not on Builder pattern, have to shift it to builder pattern.